server/channels/api4/job_test.go

```diff
@@ -261,18 +275,25 @@ func TestDownloadJob(t *testing.T) {
 	// as a system admin, we should get a not found status.
 	_, err = th.App.Srv().Store().Job().Save(job)
 	require.NoError(t, err)
-	defer th.App.Srv().Store().Job().Delete(job.Id)
+	defer func() {
+		_, delErr := th.App.Srv().Store().Job().Delete(job.Id)
+		require.NoError(t, delErr, "Failed to delete job %s", job.Id)
+	}()	
```

Comment:
The file is not `go fmt`ed
```suggestion
	}()
```